Assets/Scripts/Lobby/LobbyManager.cs

```diff
@@ -0,0 +1,9 @@
+using Unity.Services.Lobbies;
+
+namespace Lobby
+{
+    public class LobbyManager
```

Comment:
This class ends up doing quite a lot; could we split it into smaller classes to better observe Single Responsibility?

Reply:
@JudeWhite Oh definitely. Even just its name tells you it's doing too many things. Whenever you are about to use the `Manager` suffix to name a class, you know you're breaking the SRP. This class could be broken up into the following classes, named so to suggest what their single responsibility is:
- `LobbyCreatingService`
- `LobbyJoiningService`
- `LobbyPollingService`